components/filter/filter.js

```diff
@@ -207,13 +214,22 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		if (dimension.loading) {
 			return html`
 				<d2l-loading-spinner></d2l-loading-spinner>
-				<div class="d2l-offscreen">${this.localize('components.filter.loading')}</div>
+				<div class="d2l-offscreen" aria-busy="true" role="alert">${this.localize('components.filter.loading')}</div>
```

Comment:
Nit: semantically this and the "empty" `<div>` might be better off a `<p>` tags. 🤷  Note that paragraph tags come with built-in top/bottom margins which may or may make your top/bottom padding unnecessary.

Reply:
Yeah that makes sense.  Looks like I still need the padding because `d2l-body-small` adds `margin: auto;`, which removes `p`'s default margins.